curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-debugging-by-building-a-random-background-color-changer/663250b42513ef5975599c49.md

```diff
@@ -24,7 +24,7 @@ Remember that you worked with a method in the Role Playing Game that rounds a nu
 You should round `darkColorsArr.length * Math.random()` down to the nearest whole number.
 
 ```js
-assert.match(getRandomIndex.toString(), /console\.log\(\s*Math\.floor\(\s*darkColorsArr\.length\s*\*\s*Math\.random\(\s*\)\s*\)\s*\)/);
+assert.match(getRandomIndex.toString(), /console\.log\(\s*Math\.floor\(\s*(darkColorsArr\.length\s*\*\s*Math\.random\(\s*\)|(Math\.random\(\s*\)\s*\*\s*darkColorsArr\.length))\s*\)\s*\)/);
```

Comment:
```suggestion
assert.match(getRandomIndex.toString(), /console\.log\(\s*Math\.floor\(\s*(darkColorsArr\.length\s*\*\s*Math\.random\(\s*\)|Math\.random\(\s*\)\s*\*\s*darkColorsArr\.length)\s*\)\s*\)/);
```

I don't think we need this second capture group.

Reply:
yes. thanks for pointing out